src/components/OcSelect.vue

```diff
@@ -42,6 +56,11 @@ export default {
       const comboBoxElement = this.$refs.select.$el.querySelector("div:first-child")
       comboBoxElement.setAttribute("aria-label", this.$gettext("Search for option"))
     },
+    setCustomizedAttrs() {
```

Comment:
This is not setting anything. It should be a computed prop (and have a different name, like `customizedAttrs` or something like that)

Reply:
Also, you should copy the `this.$attrs` first before modifying.